### DISCUSSION TITLE
Vistor placeholders and more

### DISCUSSION THREAD

**Discussion by LuluxClient** [BG-Software-LLC/SuperiorSkyblock2#898] [Implemented Feature Requests] [answered]
it will be nice to have a placeholder to get the island name where you are and the visited island name, with that we can add it on the scoreboard and have more info like visitor island rate, island level etc ... It's a must have and bentobox has this so why not this one c:

**Comment by OmerBenGera** [comment, marked as answer]
Whats wrong with adding "_location_" after the superior? It doesnt do the job?

**Reply by LuluxClient** [reply]
> Whats wrong with adding "_location_" after the superior? It doesnt do the job?

wym by adding "location" after the superior ? I checked the wiki and couldn't find info related to that,

also while we are at it, it would be cool to have a feature to disable some menus from superior since some of them are glitched / missing some features that can be done in deluxe menu

**Reply by LuluxClient** [reply]
> > Whats wrong with adding "_location_" after the superior? It doesnt do the job?
> 
> wym by adding "location" after the superior ? I checked the wiki and couldn't find info related to that,
> 
> also while we are at it, it would be cool to have a feature to disable some menus from superior since some of them are glitched / missing some features that can be done in deluxe menu

oh nvm I think I found the part of the wiki for the location stuff I'll try some stuff and update you about it.

**Reply by OmerBenGera** [reply]
> > Whats wrong with adding "_location_" after the superior? It doesnt do the job?
> 
> wym by adding "location" after the superior ? I checked the wiki and couldn't find info related to that,
> 
> also while we are at it, it would be cool to have a feature to disable some menus from superior since some of them are glitched / missing some features that can be done in deluxe menu

For example, "superior_island_level" will return the level of your island while "superior_location_island_level" will return the level of the island you are standing in.

**Comment by LuluxClient** [comment]
Hey thanks it's good also a question, in /is top how could I make it so the player heads aren't steve heads ?

**Reply by OmerBenGera** [reply]
> Hey thanks it's good also a question, in /is top how could I make it so the player heads aren't steve heads ?

You probably dont have a skins player installed on your spigot server.

**Reply by LuluxClient** [reply]
> > Hey thanks it's good also a question, in /is top how could I make it so the player heads aren't steve heads ?
> 
> 
> 
> You probably dont have a skins player installed on your spigot server.

Where could I get that ?

**Reply by OmerBenGera** [reply]
> > > Hey thanks it's good also a question, in /is top how could I make it so the player heads aren't steve heads ?
> > 
> > 
> > You probably dont have a skins player installed on your spigot server.
> 
> Where could I get that ?

SkinsRestorer and ChangeSkin are supported

**Reply by LuluxClient** [reply]
> > > > Hey thanks it's good also a question, in /is top how could I make it so the player heads aren't steve heads ?
> 
> > > 
> 
> > > 
> 
> > > You probably dont have a skins player installed on your spigot server.
> 
> > 
> 
> > Where could I get that ?
> 
> 
> 
> SkinsRestorer and ChangeSkin are supported

Thanks !
